### PR TITLE
Remove lots of tagging-related code

### PR DESCRIPTION
**Pull request by tijmenb**
Individual commits tell the complete story. Most important change here is that we no longer sync organisations from Whitehall and publishing-api, because nothing is using this information.

These PRs need to be merged first:
- https://github.com/alphagov/govuk-app-deployment/pull/60
- https://github.gds/gds/alphagov-deployment/pull/1259

### CODE REVIEW

File: app/controllers/artefacts_controller.rb [hunk 1]
```diff
@@ -48,7 +48,6 @@ def new
   end
 
   def edit
```
Comment (jackscotti) [inline app/controllers/artefacts_controller.rb]: the method stays?

File: db/migrate/20140604100157_tag_organisations_to_travel_advice_and_business_support.rb [hunk 2]
```diff
@@ -1,74 +0,0 @@
-class TagOrganisationsToTravelAdviceAndBusinessSupport < Mongoid::Migration
```
Comment (jackscotti) [inline db/migrate/20140604100157_tag_organisations_to_travel_advice_and_business_support.rb]: Is it okay to delete migrations?

**Review by jackscotti — COMMENTED** [review]
Just a couple of comments, but looks beautiful. 👌

File: db/migrate/20140604100157_tag_organisations_to_travel_advice_and_business_support.rb (hunk 2 above)
Comment (tijmenb) [inline db/migrate/20140604100157_tag_organisations_to_travel_advice_and_business_support.rb]: Yeah, if they're data-only migrations.

File: app/controllers/artefacts_controller.rb (hunk 1 above)
Comment (tijmenb) [inline app/controllers/artefacts_controller.rb]: Yes, you can still edit it in this app. It just doesn't need a special ivar set (`@artefact` is set in a before action method).

**Comment by jackscotti** [comment]
:ok_man:

**Pull request merged**